### DISCUSSION TITLE
High Idle Cpu Linux

### DISCUSSION THREAD

**Discussion by CrypticEntity** [ppy/osu#15521] [Q&A] [answered]
Htop and terminal output
https://imgur.com/a/GY1Oszd

In windows on the same computer it uses about 10% cpu usage
i7-11370H

**Comment by smoogipoo** [comment]
- Does this happen immediately after starting osu!?
- Does it happen at vsync frame limiter?

**Reply by peppy** [reply]
Could also check whether the audio volume is dimming. Probably going to be focused handled wrong by whatever window manager is on the plate today.

**Reply by CrypticEntity** [reply]
During the load screen it stays at 30%ish then goes to 70 after in intro happens.
It happens at vsync when focused (which is 144) and when the windows in not focused where it drops to 60
Not sure what ppy is saying but the ingame volume is max and the WM is a barebones minima,l openbox install

**Comment by CrypticEntity** [comment, marked as answer]
It was my problem, I didn't install the mesa driver correctly.
